foxy/distribution.yaml

```diff
@@ -310,6 +310,21 @@ repositories:
       url: https://github.com/ros2/rcl_logging.git
       version: master
     status: maintained
+  rcpputils:
+    doc:
+      type: git
+      url: https://github.com/ros2/rcpputils.git
+      version: 0.3.0
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/ros2-gbp/rcpputils-release.git
+      version: 0.3.0-2
+    source:
+      type: git
+      url: https://github.com/ros2/rcpputils.git
+      version: 0.3.0
```

Comment:
rosdistro doesn't like tags for source entries. Please change this to the branch containing the tag (`master` I presume).